### PR TITLE
Add nextjs unplugin example

### PR DESCRIPTION
**Pull request by Mokshit06**

### CODE REVIEW

File: integration/unplugin/examples/nextjs/components/button.civet [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export function Button()
+  return
```
Comment (edemaine) [inline integration/unplugin/examples/nextjs/components/button.civet:2]: I'd maybe omit `return`

**Review by edemaine — APPROVED** [review]
Nice! Should we link to this from `integration/unplugin/README.md`? Along with Astro?

**Comment by edemaine** [comment]
Oops, CI build (type checking) is failing.

**Comment by Mokshit06** [comment]
I think this type error is occurring uses `React.*` types and imports a few things from `next`: packages which are part of `examples/nextjs/package.json` which we don't install during build step.

**Comment by edemaine** [comment]
Could we just ignore these in `tsconfig`?

**Pull request merged**